src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.diff.sql

```diff
@@ -83,5 +124,121 @@ AS
     COMMIT TRANSACTION
 GO
 
-COMMIT TRANSACTION
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetExtendedQueryTagErrors
+--
+-- DESCRIPTION
+--     Gets the extended query tag errors by tag path.
+--
+-- PARAMETERS
+--     @tagPath
+--         * The TagPath for the extended query tag for which we retrieve error(s).
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetExtendedQueryTagErrors (@tagPath VARCHAR(64))
+AS
+	SET NOCOUNT     ON
+	SET XACT_ABORT  ON
+	BEGIN TRANSACTION
+
+
+    DECLARE @tagKey INT        
+
+	SELECT @tagKey = TagKey
+	FROM dbo.ExtendedQueryTag WITH(XLOCK)
+	WHERE dbo.ExtendedQueryTag.TagPath = @tagPath
+
+	-- Check existence
+	IF (@@ROWCOUNT = 0)
+		THROW 50404, 'extended query tag not found', 1 
+	ELSE
+		SELECT * FROM dbo.ExtendedQueryTagError
+		WHERE TagKey = @tagKey
+
+	COMMIT TRANSACTION
+GO
+
+/***************************************************************************************/
+-- STORED PROCEDURE
+--    DeleteExtendedQueryTagErrors
+--
+-- DESCRIPTION
+--    Delete errors for specified extended query tag
+--
+-- PARAMETERS
+--     @tagPath
+--         * The extended query tag path
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.DeleteExtendedQueryTagErrors (@tagPath VARCHAR(64))
+AS
+	SET NOCOUNT     ON
+	SET XACT_ABORT  ON
+	BEGIN TRANSACTION
+
+    DECLARE @tagKey INT        
+
+	SELECT @tagKey = TagKey
+	FROM dbo.ExtendedQueryTag WITH(XLOCK)
+	WHERE dbo.ExtendedQueryTag.TagPath = @tagPath
+
+	-- Check existence
+	IF (@@ROWCOUNT = 0)
+		THROW 50404, 'extended query tag not found', 1 
```

Comment:
Similar question as above, but if a user tries to delete something that doesn't exist, is that a 404?